### PR TITLE
Fix order table RTL alignment

### LINKED ISSUE #2818: Table alignment for right/left languages
The orders table should have better alignment for right/left languages. 

orders table

### PR DESCRIPTION
**Pull request by kieha**
Resolves #2818. Also, adds translations for order table headers.

### How to test
- Change language to an RTL one 
- Visit the orders dashboard
- Observe that the table and its elements are properly aligned

P.S. `reaction reset` for the translations on the order table headers to take effect.

### CODE REVIEW

**Comment by rymorgan** [comment]
cart_completed

**Comment by rymorgan** [comment]
cart_completed2

**Comment by kieha** [comment]
@rymorgan ready for another look.

**Comment by rymorgan** [comment]
Great, took a look at the spacing, and that's fixed.

**Comment by kieha** [comment]
@spencern this is ready for a review.

**Comment by kieha** [comment]
@spencer you need to `reaction reset` for the translations to take effect. The order table header names were hard-coded before, so I added translations for them.

File: private/data/i18n/en.json [hunk 1]
```diff
@@ -292,7 +292,18 @@
         "orderSetToState": "{{orderNumber}} {{orderText}} set to {{status}}",
         "skippedBulkOrdersAlert": "You've requested that {{selectedOrders}} {{orderText}} be set to the '{{status}}' status, but {{numberOfSkippedOrders}} of these orders {{skippedOrdersText}} not in the '{{skippedState}}' state and would skip all steps leading up to the '{{status}}' state. Are you sure you want to do this?",
         "bulkOrdersRegressionAlert": "You're about to regress {{ordersToRegress}} {{orderText}} back to the '{{status}}' state. Are you sure you want to continue with this operation?",
-        "paymentCaptureSuccess": "Payments successfully captured"
+        "paymentCaptureSuccess": "Payments successfully captured",
+        "table": {
```
Comment (spencern) [inline private/data/i18n/en.json]: We need to be putting these i18n definitions in the `/server/i18n` directory within the plugin and not in the project wide `/private/data/i18n` file. I'll fix this one, but let's make sure that from here on out, the i18n defs are in the plugins directory whenever they are used by a plugin

**Comment by spencern** [comment]
@kieha This is pretty much ready to merge, but the latest marketplace merge introduced a conflict. Can you resolve this when you get a chance?

**Comment by kieha** [comment]
@spencern conflicts fixed.

**Comment by spencern** [comment]
I'm still seeing one more conflict here in the browser @kieha

**Comment by kieha** [comment]
@spencer fixed. Good to go.

**Comment by spencer** [comment]
You're @-ing me not the Spencer you intended. (I've been ignoring it, but thought I should probably point it out)

**Pull request merged**